### PR TITLE
docs: fix exit link in CS and TOC, shift command explanation to match TOC

### LINKED ISSUE #227: [docs-ug] exit command is not linking to the exit command section on the toc
[docs-ug] exit command is not linking to the exit command section on the toc

### PR DESCRIPTION
**Pull request by yusufaine**
Closes #227

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -209,6 +209,13 @@ Example:
 
 [<sub>return to table of content</sub>](#table-of-content)
 
+### Exiting the program: `exit`
+Type exit to quit the program, alternatively you can exit by clicking on the :X button at the top right hand corner of the program. <br>
```
Comment (blaketingyu) [inline docs/UserGuide.md:213]: Don't really unds how the linking thing works but if it works it works hehe

**Review by blaketingyu — APPROVED** [review]
LGTM! Thanks for the fix!

File: docs/UserGuide.md (hunk 1 above)
Comment (yusufaine) [reply docs/UserGuide.md:213]: @blaketingyu i used intelliJ's auto-complete, but it should follow the whole string excluding symbols

**Review by aidanaden — APPROVED** [review]
LGTM! no idea how these links work tbh - how does #exiting-the-program-exit link to "### Exiting the program: `exit`"??? mysterious markdown tings

so close to finishiNG!!! 🍝

**Pull request merged**